app/models/concerns/slackable.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Slackable
+  extend ActiveSupport::Concern
+
+  included do
+    attr_accessor :skip_slack_invite
+
+    after_commit :send_slack_invite, on: :create,
```

Comment:
On the fence about this due to volume. Might be better if we add a trigger for this e.g. a "Connect Slack" button in the UI instead of full-send inviting every new signup to a private Slack channel. But maybe we can try it.